events.yaml

```diff
@@ -114,6 +129,14 @@
   country: Switzerland
   category: Competition
 
+- title: ROSCon China 2024
+  link: http://roscon.roseducation.org.cn/2024/en/
+  start_date: 2024-11-2
+  end_date: 2024-11-3
+  city: Odense
```

Comment:
Thanks Kimberly! I'm on mobile so can't suggest the changes reliably but these fields seem to be from Roscon 2024?

Reply:
ah woops, classic copy past. I'll fix it, thanks!